### DISCUSSION TITLE
What volumes need backing up?

### DISCUSSION THREAD

**Discussion by cortopy** [goharbor/harbor#20708] [Q&A] [answered]
The [documentation](https://goharbor.io/docs/main/administration/backup-restore/) explains how to do backup and restore using Velero. However, I don't have that in my kubernetes cluster and I was wondering what needs to be backed up. In terms of file persistence, there are four volumes that a typical installation would use:

- Postgresql storage: This is a no-brainer but there are many tools to to a db dump from postgres.
- Redis: Does this need backing up? If it's ony used for caching and sessions, I don't mind losing that.
- Harbor-jobservice: What happens if the database is restored but not this one?
- Trivy: I don't mind if I lose the security scans, but I'm worried they are referenced somewhere in postgres and things may break

Would it be OK to just backup postgres for disaster recovery?

**Comment by wy65701436** [comment, marked as answer]
Only the storage of your artifacts and database are required for backing up. And if you would like to keep the log of executed jobs, like replication, GC and etc, you can also backup the storage of your jobservice.
